MovingCameraForegroundEstimetor/ForegroundEstimetor.py

```diff
@@ -65,6 +65,32 @@ def __init__(self, num_models: int = 2, block_size: int = 4, var_init: float = 2
 
         self.compile()
 
+    def update(self, num_models: int = 2, block_size: int = 4, var_init: float = 20.0*20.0, var_trim: float = 5.0*5.0,
```

Comment:
This can be dangerous when you want to update specific parameters. remove the default values and maybe consider to give a dict like kwargs